pkg/odo/cli/pipelines/bootstrap.go

```
@@ -77,9 +103,127 @@ func (io *BootstrapParameters) Complete(name string, cmd *cobra.Command, args []
 		identifier := factory.NewDriverIdentifier(factory.Mapping(host, io.PrivateRepoDriver))
 		factory.DefaultIdentifier = identifier
 	}
+
+	// ask for sealed secrets only when default is absent
+	flagset := cmd.Flags()
+	if flagset.NFlag() == 0 {
+		err := initiateInteractiveMode(io)
+		if err != nil {
+			return err
+		}
+	} else {
+		err := nonInteractiveMode(io)
+		if err != nil {
+			return err
+		}
+	}
+
+	io.Overwrite = true
```

`overwrite` is always true in non-interactive mode. It should throw an error when the directory already exists